pom.xml

```
@@ -42,15 +42,18 @@
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
 
     <!-- jenkins versions -->
-    <jenkins.version>2.426.3</jenkins.version>
-    <bom.artifactId>bom-2.426.x</bom.artifactId>
-    <bom.version>3208.vb_21177d4b_cd9</bom.version>
+    <jenkins.version>2.472</jenkins.version>
+    <bom.artifactId>bom-weekly</bom.artifactId>
```

do not do this:
https://github.com/jenkinsci/bom#depending-on-bom-weekly
```suggestion
    <bom.artifactId>bom-2.462.x</bom.artifactId>
```